api/.nycrc.json

```
@@ -0,0 +1,13 @@
+{
+  "all": true,
+  "reporter": "html",
+  "include": [
+    "lib/**/*.js"
+  ],
+  "temp-dir": "/tmp/pix-api-test-coverage/tmp",
+  "report-dir": "/tmp/pix-api-test-coverage/report",
+  "exclude": [
+    "**/.eslintrc.js",
+    "./lib/infrastructure/validate-environement-variables.js"
+  ]
```

Juste changement de l'ordre des propriétés pour avoir maintenant `include` et `exclude` à côté :
 
```suggestion
  "include": [
    "lib/**/*.js"
  ],
  "exclude": [
    "**/.eslintrc.js",
    "./lib/infrastructure/validate-environement-variables.js"
  ],
  "temp-dir": "/tmp/pix-api-test-coverage/tmp",
  "report-dir": "/tmp/pix-api-test-coverage/report"
```

---

En effet. C'est corrigé